README.md

```diff
@@ -113,16 +125,17 @@ icon and then `Publish Data`.
 
 Fill in the form for publishing and submit. 
 For simplicity, you can enter any name for group, artifact and version.
-Use the URI of the file we prepared for publishing (`https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md`) 
+Use the URI of the file we prepared for publishing [here](https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md) 
 in the `Files` section.  
 
 After publishing the data should be visible on  `account icon -> My Account -> Data tab`.
 
 ℹ️ Notes:
-* See more on how to organise your data [here](https://dbpedia.gitbook.io/databus/model), there you can find detailed explanations and our suggestions on structuring your datasets.
-* For automated metadata uploads see the [API](https://dbpedia.gitbook.io/databus/usage/api)
+- See more on how to organise your data [here](https://dbpedia.gitbook.io/databus/model), there you can find detailed explanations and our suggestions on structuring your datasets.
```

Comment:
Doesn't removing bullet points make it less emphasised? I would keep them, I think...

Reply:
Actually it was like a "*" I have modified it as a bullet by giving : "-"